ccswbs/testing
# Add code documentation

## Test Procedure
1. Checkout branch
2. Run `npm install` and `npm install --dev`
3. Run `npm run docs`

Just check to make sure the documentation is properly generated.
